<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartWindow</class>
 <widget class="QMainWindow" name="StartWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Capablanca</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="layout_info">
      <item row="1" column="2">
       <widget class="QLabel" name="label_team">
        <property name="text">
         <string>FTC team #6424</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_author">
        <property name="text">
         <string>Created by 'Oly Cow Robotics</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" rowspan="2">
       <widget class="QLabel" name="label_capablanca">
        <property name="text">
         <string>Capablanca</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" rowspan="2">
       <spacer name="spacer_info">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>864</width>
         <height>201</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <spacer name="spacer_seasons_top">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>65</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="layout_seasons">
          <item>
           <widget class="QPushButton" name="button_half_pipe_hustle">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>2005-2006
Half-Pipe Hustle</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_hangin_a_round">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>2006-2007
Hangin'-A-Round</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_quad_quandary">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>2007-2008
Quad Quandary</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_face_off">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>2008-2009
Face Off!</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_hot_shot">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>2009-2010
Hot Shot!</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_get_over_it">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>2010-2011
Get Over It!</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_bowled_over">
            <property name="text">
             <string>2011-2012
Bowled Over!</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_ring_it_up">
            <property name="text">
             <string>2012-2013
Ring It Up!</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_block_party">
            <property name="text">
             <string>2013-2014
Block Party!</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_nuclear_fallout">
            <property name="text">
             <string>2014-2015
Nuclear Fallout!</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="spacer_seasons_bottom">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>64</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="layout_tools">
      <item row="0" column="2" rowspan="2">
       <widget class="Line" name="separator">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="button_help">
        <property name="text">
         <string>Help</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/icons/icons/appbar.book.png</normaloff>:/icons/icons/appbar.book.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="button_settings">
        <property name="text">
         <string>Settings</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/icons/icons/appbar.settings.png</normaloff>:/icons/icons/appbar.settings.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="0" rowspan="2">
       <widget class="QPushButton" name="button_score">
        <property name="text">
         <string>Score</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" rowspan="2">
       <widget class="QPushButton" name="button_replay">
        <property name="text">
         <string>Replay...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QPushButton" name="button_about">
        <property name="text">
         <string>About</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/icons/icons/appbar.info.png</normaloff>:/icons/icons/appbar.info.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="button_report_bugs">
        <property name="text">
         <string>Report Bugs</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/icons/icons/appbar.bug.png</normaloff>:/icons/icons/appbar.bug.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>button_half_pipe_hustle</tabstop>
  <tabstop>button_hangin_a_round</tabstop>
  <tabstop>button_quad_quandary</tabstop>
  <tabstop>button_face_off</tabstop>
  <tabstop>button_hot_shot</tabstop>
  <tabstop>button_get_over_it</tabstop>
  <tabstop>button_bowled_over</tabstop>
  <tabstop>button_ring_it_up</tabstop>
  <tabstop>button_block_party</tabstop>
  <tabstop>button_nuclear_fallout</tabstop>
  <tabstop>button_score</tabstop>
  <tabstop>button_replay</tabstop>
  <tabstop>button_settings</tabstop>
  <tabstop>button_help</tabstop>
  <tabstop>button_report_bugs</tabstop>
  <tabstop>button_about</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
